workitem.go

```diff
@@ -114,6 +114,42 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	})
 }
 
+// Reorder reorders the workitem
+func (c *WorkitemController) Reorder(ctx *app.ReorderWorkitemContext) error {
+	return application.Transactional(c.db, func(appl application.Application) error {
+		var dataArray []*app.WorkItem2
+		if ctx.Payload == nil || ctx.Payload.Data == nil {
+			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(errors.NewBadParameterError("data", nil))
+			return ctx.NotFound(jerrors)
+		}
+		for i := 0; i < len(ctx.Payload.Data); i++ {
+			wi, err := appl.WorkItems().Load(ctx, *ctx.Payload.Data[i].ID)
+			if err != nil {
+				jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrNotFound(fmt.Sprintf("Error reordering work item: %s", err.Error())))
```

Comment:
Please shorten these two lines by using: 

```go
return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "Failed to reorder work item"))
```

The `errs` package is an alias and you can import it like so:

```go
import (
   errs "github.com/pkg/errors"
)
```